_news/2019-09-17-deprecate-python-installer.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Terraform: Official Installation Path of Forseti Security
+author: Hannah Shin
+---
+
+Forseti Community,
+
+We’ve heard from many of our users that Forseti’s Deployment Manager-based 
+upgrade path was slow and inefficient, so we're focusing on delivering a 
+faster, simpler, and more consistent experience with Terraform. 
+
+Starting with Forseti Security v2.22.0, Forseti will support Terraform as the 
+official installation path. The Python based Deployment Manager installer will 
+be deprecated effective October 3, 2019, and will no longer be available as an 
```

Comment:
nit: how about moving the date `October 3, 2019` next to `v2.22.0` earlier in this paragraph?  seems to flow better